### PR TITLE
[HER-172] Update the design of the marketing section

### PR DESCRIPTION
**Pull request by ConstantinescuAndrei**
## Description

Update the design of the marketing section based of the following link:
- [Desktop](https://xd.adobe.com/view/f1281b54-d458-42c1-b28b-7dcca2c23b31-a47b/specs/?fbclid=IwAR3x9zSAwHJJIrQaGxIQvAKjAwys50dFidZfz7ZnOpH4-rzQX-W-kQO4LVo)
- [Tablet](https://xd.adobe.com/view/c08e8b21-5cdf-469b-9e66-2c500a5ec043-95dd/?fbclid=IwAR1x4Wn89kjsO0ZivtrxqBztZvh67ubeCMJGVc8eMmivnjwBRwteIr7zP20)
- [Smartphone](https://xd.adobe.com/view/c5fc8477-17c8-4014-872e-6e6575603d84-90af/?fbclid=IwAR2aCw3az4LwYWh4oYOW08rfN_9r8F8xh8u98QghtsQUYmHNQoKX8QUCAt4)

## Related
[HER-172](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-172)

### CODE REVIEW

File: src/home/marketing/list/list-link.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Link } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useLinkStyle } from 'home/marketing/list/link-style'
+import LinkIcon from 'home/heading-section/link/link_black_24dp.svg'
+
+export const ListLink = ({ link }) => {
+  console.log(link)
```
Comment (miutamihai) [inline src/home/marketing/list/list-link.jsx]: Remove this

File: src/home/marketing/styles/item-desktop-style.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { makeStyles } from '@material-ui/core'
+
+const root = { paddingTop: '0%' }
```
Comment (miutamihai) [inline src/home/marketing/styles/item-desktop-style.js]: We can remove the `%` sign here.

**Review by miutamihai — CHANGES_REQUESTED** [review]

File: src/home/marketing/list/list-link.jsx (hunk 1 above)
Comment (ConstantinescuAndrei) [reply src/home/marketing/list/list-link.jsx]: done

File: src/home/marketing/styles/item-desktop-style.js (hunk 2 above)
Comment (ConstantinescuAndrei) [reply src/home/marketing/styles/item-desktop-style.js]: done

**Review by miutamihai — DISMISSED** [review]

**Review by miutamihai — APPROVED** [review]

**Pull request merged**